dms_field/readme/USAGE.md

```diff
@@ -0,0 +1,4 @@
+#\. Go to the form view of an existing partner and click on the "DMS"
+tab icon, a hierarchy of folders and files linked to that record will be
+created. \#. Create a new partner. A hierarchy of folders and files
+linked to that record will be created.
```

Comment:
![image](https://github.com/user-attachments/assets/517f3a95-fb5a-4a1f-b774-21a4744beb0d)
Shouldn't it be a list?